custom_components/tuya_local/devices/human_presence_detector.yaml

```diff
@@ -0,0 +1,139 @@
+name: human presence detector
+products:
+  - id: e833v6jexwfkjrij
+    name: RTCZ-03 human presence sensor
+primary_entity:
+  entity: binary_sensor
+  class: occupancy
+  dps:
+    - id: 1
+      type: boolean
+      name: sensor
+      mapping:
+        - dps_val: peaceful
+          value: false
+        - dps_val: presence
+          value: true
+        - dps_val: small_move
+          value: true
+        - dps_val: large_move
+          value: true
+        - dps_val: none
+          value: false
+    - id: 1
+      type: string
+      name: raw_state
+secondary_entities:
+  - entity: binary_sensor
+    class: motion
+    dps:
+      - id: 1
+        type: boolean
+        name: sensor
+        mapping:
+          - dps_val: peaceful
+            value: false
+          - dps_val: presence
+            value: false
+          - dps_val: small_move
+            value: true
+          - dps_val: large_move
+            value: true
+          - dps_val: none
+            value: false
+  - entity: sensor
+    class: distance
+    dps:
+      - id: 101
+        type: integer
+        name: sensor
+        unit: cm
+  - entity: sensor
+    class: illuminance
+    dps:
+      - id: 102
+        type: integer
+        name: sensor
+        unit: lx
+  - entity: number
+    name: clear delay
+    dps:
+      - id: 103
+        type: integer
+        name: value
+        range:
+          min: 0
+          max: 28799
+        unit: s
+  - entity: light
+    category: config
+    name: indicator light
+    dps:
+      - id: 104
+        type: boolean
+        name: switch
+  - entity: number
+    category: config
+    name: Maximum detection distance
+    dps:
+      - id: 107
+        type: integer
+        name: value
+        range:
+          min: 0
+          max: 840
+        unit: cm
+  - entity: number
+    category: config
+    name: Minimum detection distance
+    dps:
+      - id: 108
+        type: integer
+        name: value
+        range:
+          min: 0
+          max: 840
+        unit: cm
+  - entity: number
+    category: config
+    name: Detection Sensitivity
+    dps:
+      - id: 111
+        type: integer
+        name: value
+        range:
+          min: 0
+          max: 10
+  - entity: number
+    category: config
+    name: Hold Sensitivity
+    dps:
+      - id: 112
+        type: integer
+        name: value
+        range:
+          min: 0
+          max: 10
+  - entity: sensor
+    category: diagnostic
+    name: hold (min)
```

Comment:
I'm not sure how this relates to the below, but probably just a single sensor with the other value as an extra attribute would be better.
If Hold time is defined by min + sec (eg 1:30 = min:1; sec:30), then make the min one the main sensor, and have seconds as an extra attribute available if users need more resolution.
If seconds and min show the same thing with different resolutions (eg 1:30 = min:1, sec:90), then make the seconds the main sensor and leave the minutes as an extra attribute just in case someone has a use for that.
In both cases, add `class: duration` for better display of values, and following HA naming convention.

Reply:
Yeah... I'm not sure why this device is made this way but its the same delay as in DP103 but this one just spits it back out in minutes and seconds in DP105 and DP106.